metricbeat/tests/system/test_autodiscover.py

```diff
@@ -86,9 +86,4 @@ def test_docker_labels(self):
         container.stop()
         self.wait_until(lambda: self.log_contains('Autodiscover stopping runner: memcached'))
 
-        output = self.read_output_json()
```

Comment:
Is this check not needed anymore?

Reply:
I removed it as redundant, but on a second thought, it makes sense to double check here, I'll add it again

Reply:
I've been fighting them a little more, hopefully they work now, without sleeps

Reply:
It worked :+1: